### PR TITLE
14. Mejorando los Locator

### PR DESCRIPTION
**Pull request by lvzh90**
14. Mejorando los Locator

### CODE REVIEW

File: src/page/AddressStep.page.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { $, ElementFinder, promise } from 'protractor';
 
 export class AddressStepPage   {
   private get proceedToCheckout(): ElementFinder {
-    return $('#center_column > form > p > button > span');
+    return $('#center_column button span');
```
Comment (germandavid85) [inline src/page/AddressStep.page.ts]: I think this has the property `name` while that is available we should prefer that one:
`[name="processAddress"]`

File: src/page/ShippingStep.page.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ export class ShippingStepPage {
   }
 
   private get proceedToCheckout(): ElementFinder {
-    return $('#form > p > button > span');
+    return $('.cart_navigation button span');
```
Comment (germandavid85) [inline src/page/ShippingStep.page.ts]: this also should have a `name` property

**Review by germandavid85 — CHANGES_REQUESTED** [review]
just a couple of comments

File: src/page/AddressStep.page.ts (hunk 1 above)
Comment (lvzh90) [reply src/page/AddressStep.page.ts]: Hecho

File: src/page/ShippingStep.page.ts (hunk 2 above)
Comment (lvzh90) [reply src/page/ShippingStep.page.ts]: Hecho

**Review by germandavid85 — APPROVED** [review]
LGTM 👍

**Pull request merged**